test/rabbit_env_SUITE.erl

```diff
@@ -18,101 +18,102 @@
 
 -include_lib("common_test/include/ct.hrl").
 -include_lib("eunit/include/eunit.hrl").
+-include_lib("kernel/include/file.hrl").
 
--export([all/0,
-         suite/0,
-         groups/0,
-         init_per_group/2,
-         end_per_group/2,
-         init_per_testcase/2,
-         end_per_testcase/2,
-         check_data_dir/1,
-         check_default_values/1,
-         check_values_from_reachable_remote_node/1,
-         check_values_from_offline_remote_node/1,
-         check_context_to_app_env_vars/1,
-         check_context_to_code_path/1,
-         check_RABBITMQ_ADVANCED_CONFIG_FILE/1,
-         check_RABBITMQ_CONFIG_FILE/1,
-         check_RABBITMQ_CONFIG_FILES/1,
-         check_RABBITMQ_DIST_PORT/1,
-         check_RABBITMQ_ENABLED_PLUGINS/1,
-         check_RABBITMQ_ENABLED_PLUGINS_FILE/1,
-         check_RABBITMQ_FEATURE_FLAGS_FILE/1,
-         check_RABBITMQ_KEEP_PID_FILE_ON_EXIT/1,
-         check_RABBITMQ_LOG/1,
-         check_RABBITMQ_LOG_BASE/1,
-         check_RABBITMQ_LOGS/1,
-         check_RABBITMQ_MNESIA_BASE/1,
-         check_RABBITMQ_MNESIA_DIR/1,
-         check_RABBITMQ_MOTD_FILE/1,
-         check_RABBITMQ_NODE_IP_ADDRESS/1,
-         check_RABBITMQ_NODE_PORT/1,
-         check_RABBITMQ_NODENAME/1,
-         check_RABBITMQ_PID_FILE/1,
-         check_RABBITMQ_PLUGINS_DIR/1,
-         check_RABBITMQ_PLUGINS_EXPAND_DIR/1,
-         check_RABBITMQ_PRODUCT_NAME/1,
-         check_RABBITMQ_PRODUCT_VERSION/1,
-         check_RABBITMQ_QUORUM_DIR/1,
-         check_RABBITMQ_UPGRADE_LOG/1,
-         check_RABBITMQ_USE_LOGNAME/1,
-         check_value_is_yes/1,
-         check_log_process_env/1,
-         check_log_context/1
-        ]).
+-compile(export_all).
```

Comment:
Please keep the list of explicitly exported functions, otherwise there is no way to know when there is dead code.

Reply:
I had wondered why the export was explicit, since in other tests, we just export all. How does this help us find dead code?